pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -167,6 +167,10 @@ class IBMQBackend(Backend):
     :type provider: Optional[IBMProvider]
     :param token: Authentication token to use the `QiskitRuntimeService`.
     :type token: Optional[str]
+    :param options: A dictionary to specify qiskit API options,
```

Comment:
If these are the only supported options, I think it makes sense to have them as distinct kwargs, rather than having to pass them in a dictionary. Alternatively, why not just allow the user to pass an arbitrary qiskit `Options` object here?

Reply:
OK. Happy to make this change.